.buildscript/configure-compose.gradle

```diff
@@ -19,7 +19,7 @@ android {
     compose true
   }
   composeOptions {
-    kotlinCompilerVersion "1.4.0-rc"
+    kotlinCompilerVersion "1.4.0"
```

Comment:
Do we still need this line?

Reply:
Tested. We can remove this.

Reply:
Yep, but it can just refer directly to `Versions.kotlin` now. Fixed.